t/lib/TestFor/App/GHPT/WorkSubmitter/AskPullRequestQuestions.pm

```diff
@@ -9,8 +9,8 @@ with 'TestRole::WithGitRepo';
 sub test_startup {
     my $self = shift;
     $self->test_skip(
-        'This test does not run on Travis because of the way it uses git when testing a PR'
-    ) if $ENV{TRAVIS};
+        'This test does not run in GitHub Actions because of the way it uses git when testing a PR'
+    ) if $ENV{GITHUB_ACTION};
```

Comment:
There's a `CI` var in GH actions and other systems that might be better to use for future flexibility.